app/controllers/api/v1/users_orders_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::UsersOrdersController < ApplicationController
+  before_action :authorize_access_request!
+  
+  def create
+    user = User.find(params[:user_id])
+    order = user.orders.create(order_params)
+    params[:items].each do |item|
```

Comment:
I could probably remove these to maybe a poro it's a pretty bloated action right now